recipes/galaxy-parsec/meta.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: galaxy-parsec
+  version: "1.0.4"
+
+source:
+  fn: galaxy-parsec-1.0.4.tar.gz
+  url: https://pypi.python.org/packages/f7/aa/5bbd266698de39c9897d58e26bac2bff714be73ac10ed5f41b8ac45213bb/galaxy-parsec-1.0.4.tar.gz
+  md5: 066e3d35f3a374095d0d90bf76214494
+
+build:
+  # noarch: python
+  preserve_egg_dir: True
+  entry_points:
+    - parsec=parsec.cli:parsec
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - click >=6.7
+    - bioblend
+    - wrapt
+    - pyyaml
+    - justbackoff
+    - xunit-wrapper
+    - future
+
+  run:
+    - python
+    - click >=6.7
```

Comment:
@erasche since you are using click. Can you please add this to the recipe:
```
extra:
  container:
    extended-base: true
```

We will not use busybox here for testing but a slightly larger container with full UTF support.